src/Components/ImagesTable/ImageBuildStatus.js

```diff
@@ -136,7 +160,37 @@ export const ImageBuildStatus = ({ imageId, isImagesTableRow }) => {
         messages[status].map((message, key) => (
           <Flex key={key} className="pf-u-align-items-baseline pf-m-nowrap">
             <div className="pf-u-mr-sm">{message.icon}</div>
-            {message.text}
+            {status === 'failure' ? (
+              <Popover
```

Comment:
This is really nice! I like the idea of the scrollable pane. Do you think we could put this in its own React component (it can still live in this file) just so that the conditional rendering logic is a bit easier to understand at first glance?